plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -258,4 +265,19 @@ public void validate()
                 !countDistinctPushdownEnabled || aggregationPushdownEnabled,
                 "Invalid configuration: pinot.aggregation-pushdown.enabled must be enabled if pinot.count-distinct-pushdown.enabled");
     }
+
+    @AssertTrue(message = "All controller URLs have some scheme")
+    public boolean validateControllerUrlsScheme()
```

Comment:
This method doesn't do any validation. It just returns whether all schemes are the same. So, rename it to `allUrlSchemesEqual`